File: Selecao I/maior.cpp
/*@max: Maior de dois números*/

#include <iostream>
#include <locale>
using namespace std;

int main(){
    setlocale(LC_ALL, "Portuguese");
    int n1, n2;
    cin >> n1;
    cin >> n2;

    if(n1 > n2){
        cout << n2 << endl;
    }else if(n2 > n1){
        cout << n2 << endl;
    }else{
        cout << n1 << endl;
    }
    return 0;
}

File: src/Selecao_II/Maior.java
/**
 * Questão: @max: Maior de dois números
 *
 * @author UxieGu1
 */

package Selecao_II;
import java.util.Scanner;

public class Maior {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        int n1 = scan.nextInt();
        int n2 = scan.nextInt();

        if(n1 > n2) {
            System.out.println(n1);
        } else {
            System.out.println(n2);
        }

        scan.close();
    }
}
